test_scripts/Polices/External_UCS_Informing_HMI/001_ATF_P_GetListofPermissions_with_appID_all_ecs_in_bound_EntitySatus_ON.lua

```diff
@@ -0,0 +1,130 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] [External UCS] SDL informs HMI about <externalConsentStatus> via GetListOfPermissions response
+-- [HMI API] GetListOfPermissions request/response
+-- [HMI API] ExternalConsentStatus struct & EntityStatus enum
+--
+-- Description:
+-- For Genivi applicable ONLY for 'EXTERNAL_PROPRIETARY' Polcies
+-- Upon GetListOfPermissions_request, SDL must inform "externalConsentStatus" setting to HMI
+--
+-- 1. Used preconditions
+-- SDL is built with External_Proprietary flag
+-- SDL and HMI are running
+-- Application is registered and activated
+-- PTU has passed successfully
```

Comment:
@maria-mih, please add additional information that ptu file is updated and which functional groups are assigned to application.